src/components/Image/Image.tsx

```diff
@@ -99,6 +99,11 @@ export class Image extends React.Component<ImageProps> {
             [classes.loaded]: hasLoadingBehavior && isLoaded,
           }),
         )}
+        style={{
+          // If fixed dimensions were passed, we set the calculated values to fit the container with the fixed image
+          width: width && calculatedDimensions.width,
+          height: height && calculatedDimensions.height,
+        }}
```

Comment:
This logic is a fix for a bug I missed out. It's relevant to the PR in order to display the components in Settings Panel correctly when passing fixed dimensions.

Now it's covered by the visuals I added